app/controllers/accounts_controller.rb

```diff
@@ -13,7 +13,7 @@ def show
       personal_key: flash[:personal_key],
       sp_session_request_url: sp_session_request_url_with_updated_params,
       sp_name: decorated_session.sp_name,
-      decorated_user: current_user.decorate,
+      user: current_user,
```

Comment:
I just wanted to chime in and say that while I appreciate the whimsy of passing a `decorated_user` to a presenter, just being able pass a `user` is so much clearer! 🎉